bin/acmefetch

```diff
@@ -321,19 +323,33 @@ sub getCertificates {
         }
         my $csrFile = makeCsr($cfg,$cert);
         eval {
-            my $certData = convertCert($acme->sign( $csrFile ),'DER',$cert->{certFormat});
-            my $certFile = $cert->{certOutput};
-            saveAway($certFile);
-            my $fh = IO::File->new( $certFile, "w" )
-                || die "Write $certFile: $!";
-            print $fh $certData;
-            $fh->close();
-            my $chainData = convertCert($acme->chain(),'DER',$cert->{chainFormat});
-            saveAway($cert->{chainOutput});
-            my $fh2 = IO::File->new( $cert->{chainOutput}, "w" )
-                || die "Write $cert->{chainOutput}: $!";
-            print $fh2 $chainData;
-            $fh2->close();
+            if ($cert->{chainOutput}){
+                my $certData = convertCert($acme->sign( $csrFile ),'DER',$cert->{certFormat});
+                my $certFile = $cert->{certOutput};
+                saveAway($certFile);
+                my $fh = IO::File->new( $certFile, "w" )
+                    || die "Write $certFile: $!";
+                print $fh $certData;
+                $fh->close();
+                my $chainData = convertCert($acme->chain(),'DER',$cert->{chainFormat});
+                saveAway($cert->{chainOutput});
+                my $fh2 = IO::File->new( $cert->{chainOutput}, "w" )
+                    || die "Write $cert->{chainOutput}: $!";
+                print $fh2 $chainData;
+                $fh2->close();
+            }
+            # no chain file wanted -> include chain in cert file
+            else {
+                my $certData = convertCert($acme->sign( $csrFile ),'DER',$cert->{certFormat});
+                my $certFile = $cert->{certOutput};
+                my $chainData = convertCert($acme->chain(),'DER',$cert->{chainFormat});
+                saveAway($certFile);
+                my $fh = IO::File->new( $certFile, "w" )
+                    || die "Write $certFile: $!";
+                print $fh $certData;
+                print $fh $chainData;
+                $fh->close();
+            }
             saveAway($cert->{keyOutput});
```

Comment:
there is quite substantial code duplication ... how about just doing:

```perl
if ($cert->{chainOutput}){
￼   $fh->close();
   saveAway($cert->{chainOutput});
   $fh = IO::File->new( $cert->{chainOutput}, "w" )
￼                     || die "Write $cert->{chainOutput}: $!";
}
```
on line 330 of the original code and replace fh2 with fh